packages/flutter/lib/cupertino.dart

```diff
@@ -8,7 +8,9 @@
 library cupertino;
 
 export 'src/cupertino/activity_indicator.dart';
+export 'src/cupertino/button.dart';
 export 'src/cupertino/dialog.dart';
 export 'src/cupertino/slider.dart';
 export 'src/cupertino/switch.dart';
 export 'src/cupertino/thumb_painter.dart';
+export 'widgets.dart';
```

Comment:
I'm not sure about this line.  It's true we do this in material.dart, which is a good reason to do it here.  However, this will make the dartdoc for Cupertino pretty overwhelming...  I'm inclined to say we should skip this for now.  We can always add it later, but once we add it, it's hard to remove because removing it will break clients.